app/models/invite_request.rb

```diff
@@ -30,15 +31,24 @@ def position
     InviteRequest.where("id <= ?", id).count
   end
 
-  #Ensure that invite request is for a new user
+  # Ensure that email is not banned
+  def check_admin_banned_list
+    return unless AdminBlacklistedEmail.exists?(email: self.email)
```

Comment:
`is_blacklisted?` would be better here to match the email canonization/simplification that the banlist does.  (It'd probably be good to have a test for the blocking of the variants. The general variants that should work are the [same as for guest comments](https://otwarchive.atlassian.net/browse/AO3-4536).)